ChatterboxBackend/Chatterbox/Controllers/UserController.cs

```diff
@@ -0,0 +1,79 @@
+﻿using Chatterbox.Models;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Chatterbox.Controllers
+{
+    [Route("api/[controller]")]
+    public class UserController:Controller
+    {
+        private readonly ChatterContext _context;
+
+        public UserController(ChatterContext context)
+        {
+            _context = context;
+        }
+
+        [HttpPost("login")]
+        public IActionResult LogMeIn([FromBody] LoginDetails details)
+        {
+            Console.WriteLine("NEW LOG",details.Email);
+            var emailCheck = _context.Users.FirstOrDefault(t => t.Email == details.Email);

```

Comment:
Again, rather use `IQueryable.Any()` to check for existence, if you're not going to use the resulting object.